libs/utils/trace_analysis.py

```diff
@@ -934,6 +930,9 @@ def getClusterFrequencyResidency(self, cluster):
         total_time = total_time.groupby(['frequency']).sum()
 
         ### Compute ACTIVE Time ###
+        # Assumption: all CPUs in a cluster run at the same frequency, i.e. the
```

Comment:
Should not we use the Trace.freq_coherency attribute to raise a warning/exception in case we call this method on a non freq_chorent platform?